Dockerfile

```diff
@@ -29,7 +29,9 @@ RUN apt-get update \
         valgrind \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-    && ln -sf /usr/bin/llvm-symbolizer-3.8 /usr/bin/llvm-symbolizer
+    && ln -sf /usr/bin/llvm-symbolizer-3.8 /usr/bin/llvm-symbolizer \
+    && python $(curl https://bootstrap.pypa.io/get-pip.py) \
```

Comment:
Is it maybe cleaner to install pip via apt-get?

Reply:
done